README.md

```diff
@@ -72,6 +72,18 @@ repository:
   # vulnerability alerts.
   enable_vulnerability_alerts: true
 
+  # Either `true` to make this repo available as a template repository or `false` to prevent it.
+  # is_template: false
+  # template_repository: false
```

Comment:
i see `is_template` listed on https://docs.github.com/en/rest/reference/repos#update-a-repository, but i dont find `template_repository` mentioned there. were both intentionally included here?

also, is there a reason that this one is commented out?

Reply:
you can see it on https://docs.github.com/en/rest/reference/repos#list-organization-repositories
> The is_template and template_repository keys are currently available for developer to preview.